roles/openshift_health_checker/library/docker_container.py

```diff
@@ -0,0 +1,2033 @@
+#!/usr/bin/python
+#
+# Copyright 2016 Red Hat | Ansible
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+# This file is a copy of https://github.com/ansible/ansible/commit/20bf02f6b96356ab5fe68578a3af9462b4ca42a5
+# It has been temporarily vendored here because of X, Y, Z.
+# We can remove this file once openshift-ansible requires ansible >= 2.x.x.x.
+# pylint: skip-file
```

Comment:
This must be at the very beginning of the module, right after the shebang, otherwise it has no effect.
https://pylint.readthedocs.io/en/latest/faq.html#how-can-i-tell-pylint-to-never-check-a-given-module

I'd also suggest that our own commentary should be the top-most thing, and everything else should be as-is in the original.